tidb-lightning/troubleshoot-tidb-lightning.md

```diff
@@ -201,3 +201,7 @@ This error occurs usually because the CSV data file does not contain a header (t
 [mydumper.csv]
 header = false
 ```
+
+### `Unknown character set`
+
+TiDB does not support all MySQL character sets. Therefore, TiDB Lightning reports this error if an unsupported character set is used when creating the table structure during an import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance.
```

Comment:
Please check if similar changes need to be made.

Reply:
Fixed in [7a3ee27](https://github.com/pingcap/docs/pull/10974/commits/7a3ee271b216780b8e016083758bc7688c4118bb)